src/db/commit.rs

```diff
@@ -539,33 +559,89 @@ pub enum LoadError {
     DuplicateIndexName(String),
 }
 
+impl ToJsValue for LoadError {
+    fn to_js(&self) -> Option<&JsValue> {
+        match self {
+            LoadError::InvalidCookieJson(_) => None,
+            LoadError::MissingCookie => None,
+            LoadError::MissingMutatorName => None,
+            LoadError::MissingMutatorArgsJson => None,
+            LoadError::MissingTyped => None,
+            LoadError::MissingMeta => None,
+            LoadError::MissingValueHash => None,
+            LoadError::UnknownMetaType => None,
+            LoadError::InvalidIndex((_, e)) => e.to_js(),
+            LoadError::DuplicateIndexName(_) => None,
+        }
+    }
+}
+
 #[derive(Debug, PartialEq)]
 pub enum ValidateIndexDefinitionError {
     MissingName,
     MissingKeyPrefix,
     MissingIndexPath,
 }
 
+impl ToJsValue for ValidateIndexDefinitionError {
+    fn to_js(&self) -> Option<&JsValue> {
+        match self {
```

Comment:
this could just return None but by using a match it will be a compile error if the enum changes